kale/worker.py

```diff
@@ -157,7 +157,7 @@ def _on_task_failed(self, message, time_remaining_sec, err,
         :param bool permanent_failure: whether this task permanently fails.
         """
         task = message.task_inst
-        logger.info(('Task failed. Task id: %s; Queue: %s; '
+        logger.debug(('Task failed. Task id: %s; Queue: %s; '
```

Comment:
Seems like this should be a warning or even error?

Reply:
So the fact that a task failed is already being logged in the `Task.handle_failure` method. The only additional information that logging it here gets you is the queue name. In practice, I don't think this is useful information and can easily be derived from the task name.

More generally IMO all of the worker hooks should be at the batch level and there should not be any task level hooks on the worker, so I'd like to de-emphasize these logs as much as possible and maybe just get rid of them in the future.